ironfish/src/rpc/routes/wallet/removeAccount.ts

```diff
@@ -49,7 +50,7 @@ router.register<typeof RemoveAccountRequestSchema, RemoveAccountResponse>(
       }
     }
 
-    await node.wallet.removeAccount(account.name)
+    await node.wallet.removeAccount(account.name, request.data.wait)
```

Comment:
This might cause the CLI to seem unresponsive -- waiting on the account data to be removed might take a long time, and no response is returned until it's finished

That might be ok, and I think the only alternative would be to create a streaming endpoint that returns some progress/status messages

Reply:
I added a notify message in `removeAccount cli` for now.